packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissible.ts

```diff
@@ -21,23 +20,65 @@ export function useInternalChipDismissible({
   const availableChipOptions = chipOptions.filter(
     chip => !selected.includes(chip.value),
   );
-  const { liveAnnounce } = useLiveAnnounce();
 
   const actions = {
-    handleChipRemove: (chip: typeof chipOptions[number]) => {
-      return () => {
-        liveAnnounce(`${chip.label} Removed`);
-        onChange(selected.filter(val => val !== chip.value));
-      };
+    handleChipRemove: (value: string) => {
+      return () => onChange(selected.filter(val => val !== value));
     },
+
     handleChipAdd: (value: string) => onChange([...selected, value]),
     handleCustomAdd: onCustomAdd,
+
     handleChipClick: (value: string) => {
       if (onClick === undefined) return;
       return (event: MouseEvent<HTMLButtonElement>) => onClick(event, value);
     },
+
     handleEmptyBackspace: () => {
-      actions.handleChipRemove(sortedVisibleChipOptions[selected.length - 1])();
+      actions.handleChipRemove(selected[selected.length - 1])();
+    },
+
+    handleWrapperKeyDown: (
+      event: KeyboardEvent<HTMLDivElement> & {
+        target: HTMLDivElement;
```

Comment:
Getting currentTarget and target confused as per our convo!  However, there should always be that add button rendered, so I am thinking that since it's always going to be a child element in the wrapper the target should always be there.